lib/config/config-initializer.js

```diff
@@ -306,6 +307,15 @@ function promptUser(callback) {
             }
         },
         {
+            type: "confirm",
+            name: "airbnbReact",
+            message: "Do you use React?",
```

Comment:
Maybe we could have a helper function like

```js
function prompt(feature) {
  return {
    type: 'confirm',
    name: feature,
    message: `Do you use ${feature}?`
  };
}
```

But that might be better as a separate enhancement rather than bundling it into this PR.